framework/src/st_things/things_stack/utils/things_ping.h

```diff
@@ -33,12 +37,39 @@ typedef enum ping_state_e {
 	PING_ALL_FLAG = 0xFFFFFFFF
 } ping_state_e;
 
+//-------------------------------------------------------------------------------
+/**
+ * @brief API for initialize OIC device keepAlive for ping resource.
+ * @return res : sucess = true, otherwise false.
```

Comment:
I think you follow doxygen format. Then, you should not use `:` at next of variable

Reply:
ok